src/components/firstProfile/interests/Interests.tsx

```diff
@@ -0,0 +1,275 @@
+      } catch (error) {
+        console.error('Error parsing JSON:', error)
+      }
+    }
+  }, [])
+
+  useEffect(() => {
+    localStorage.setItem('selectedLanguages', JSON.stringify(selectedLanguages))
+  }, [selectedLanguages])
+  const saveDataToLocalStorageInterests = () => {
+    const selectedInterests = interestsData.map((item) => item.selectedItems)
+    localStorage.setItem('interestsData', JSON.stringify(selectedInterests))
+  }
+
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setAboutMe(event.target.value)
+    localStorage.setItem('aboutMe', event.target.value)
+  }
+
+  return (
+    <Box className={classes.mainBox}>
+      <Box className={classes.titleContainer}>
+        <Typography className={classes.title}>Lifestyle</Typography>
+      </Box>
+      <Box className={classes.itemContainer}>
+        {interestsData.map((data, index) => (
+          <InterestsItem
+            key={index + data.title}
+            data={data}
+            index={index}
+            onToggle={(isOpen) => {
+              setIsOpenTabPointer(isOpen ? data.title : '')
+              setInterestsData((prev) => {
+                const newInterestsData = [...prev]
+                newInterestsData[index].isOpen = isOpen
+                saveDataToLocalStorageInterests()
+                return newInterestsData
+              })
+            }}
+            onSelectedItems={(selectedItems) => {
+              setInterestsData((prev) => {
+                const newInterestsData = prev
+                newInterestsData[index].selectedItems = selectedItems
+                saveDataToLocalStorageInterests()
+                return newInterestsData
+              })
+            }}
+          />
+        ))}
+        <LanguageItem
+          onSetSelectedLanguages={setSelectedLanguages}
+          selectedLanguages={selectedLanguages}
+        />
+      </Box>
+      <Box className={classes.titleContainer}>
+        <Typography className={classes.title}>About me</Typography>
+      </Box>
+      <Box className={classes.aboutMeContainer}>
+        <TextField
+          id="aboutMe"
+          type="text"
+          placeholder="Add about me..."
+          value={aboutMe}
+          onChange={onChange}
+          multiline
+          rows={6}
+          InputProps={{
+            classes: {
+              root: classes.textareaRoot,
+            },
+            inputProps: {
+              maxLength: 1000,
+            },
+          }}
+          variant="outlined"
+        />
+      </Box>
+    </Box>
+  )
+}
+
+export default Interests
+
+const useStyles = makeStyles()(() => {
+  return {
+    mainBox: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      maxWidth: '540px',
+      width: '540px',
+      justifyContent: 'center',
+      '@media (max-width: 600px)': {
+        maxWidth: '280px',
+        width: '280px',
+      },
+    },
+    chipContainer: {
```

Comment:
It looks like this style, `arrowBtn`, `arrowSvg` and probably several others are not more needed in this component. Please check it out.

Reply:
+done